components/src/core/user-menu/user-menu.component.scss

```diff
@@ -39,22 +39,45 @@ mat-card.pxb-user-menu-overlay {
 }
 
 .pxb-user-menu-header mat-toolbar {
-    border-top-left-radius: 4px !important;
-    border-top-right-radius: 4px !important;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
     .pxb-drawer-header-title {
         font-size: 1rem;
     }
     .pxb-drawer-header-subtitle {
         font-size: 0.875rem;
     }
 }
+.pxb-user-menu-bottomsheet .pxb-user-menu-header mat-toolbar {
+    border-top-left-radius: 0px;
+    border-top-right-radius: 0px;
+}
 
-.pxb-user-menu-overlay-backdrop {
-    background-color: unset;
+.cdk-overlay-backdrop-showing {
+    &.pxb-user-menu-overlay-backdrop {
+        background-color: unset;
+    }
+    &.pxb-user-menu-bottomsheet-backdrop {
+        background-color: #424e54;
```

Comment:
Should we be importing the colors package and using gray 900 here instead of the hardcoded value? Do we have a common practice/guideline for this?

Reply:
Right now our component library doesn't have a dependency on the colors package and this hex code will be overwritten by our theme.  I just picked a reasonable default as a placeholder.  The backdrop above is a good example - without this default-override or our theme-overrides (TODO) the backdrop color is `unset` so I just looked up the hex code for black[500].  

Since our practice so far has been to rely on our themes package to provide color overrides; I don't think this inline-hex code is an issue since it'll be overwritten anyways.  

On the otherhand, if we add the @pxblue/colors package, I think we would start to more commonly set our default colors in the component library instead of the themes.  This would could cause an issue down the road for teams using non-blue themed palettes.   We _could_ add the colors package, but we'd then we'd need to be extra attentive to make sure we don't put any hard-coded values that will go unnoticed in a blue/bluedark theme, but be look out-of-place in a non-blue theme. 

If we could access the applied-theme inside our component library repo, I'd suggest moving our component styles into the comp library repo, but unfortunately we can't do that.    For now I'd say we just add the placeholder styles so non-themed components looks at least decent.